src/vs/workbench/contrib/notebook/browser/diff/diffComponents.ts

```diff
@@ -1644,7 +1644,7 @@ export class ModifiedElement extends AbstractElementRenderer {
 			{
 				updateInfoRendering: () => renderSourceEditor(),
 				checkIfModified: (cell) => {
-					return cell.modified?.textModel.getValue() !== cell.original?.textModel.getValue() ? { reason: undefined } : false;
+					return cell.modified?.textModel.getTextBufferHash() !== cell.original?.textModel.getTextBufferHash() ? { reason: undefined } : false;
```

Comment:
A helper function would be nice here since you repeat it multiple times

Reply:
Will do that in debt